packages/origin-247-certificate/src/offchain-certificate/offchain-certificate.service.ts

```diff
@@ -43,26 +44,26 @@ export class OffchainCertificateService<T = null> {
             toTime: Math.round(params.toTime.getTime() / 1000)
         } as IIssueCommand<T>;
 
-        const savedCommand = await this.certCommandRepo.create({ payload: command });
-        this.validateCommand(command); // this will probably throw
+        const savedCommand = await this.certCommandRepo.save({ payload: command });
         const event = new CertificateIssuedEvent(this.generateInternalCertificateId(), command);
```

Comment:
So regarding this `generateInternalCertificateId` - why do we use date get time for that? It's not unique enough, that's the first thing. I guess that's because you wanted to keep number? I do agree with this idea, but we have to make it more unique. Some research may be required.

Also, this will still produce breaking change, because previously we were using `int4` for storing integers in postgres, and now storing `.getTime` will require storing it as `int8` i think.